### DISCUSSION TITLE
Feature idea: Notebook (or similar) organizational options

### DISCUSSION THREAD

**Discussion by renrosz** [flxzt/rnote#217] [Ideas]
First, thank you so much for developing this.

I am hoping to migrate over from OneNote and so far this program has everything I need, except for organizational options. (Unless I just haven't figured out how to do that??)

I loved being able to make a notebook, then have dividers, and then have tabs, and then pages of content - organizational dreams!
I use the note-taking software for medical school so I have A TON of work and information to keep organized and having so much flexibility with OneNote, in that regard, was fantastic. 

I'm not a dev, so I have no idea if this is super complicated to do, but figured it might be a feature worth exploring.

Absolutely incredible work so far with this app! I love it!

**Comment by flxzt** [comment]
Hi, thanks! You might want to check out the beta, which introduces workspace folders in an additional sidebar. Tabs will come too eventually.

[This Guide]( https://discourse.flathub.org/t/how-to-use-flathub-beta/2111) explains how to install the flathub beta repository.

**Comment by mizhozan** [comment]
I also have a organized notes, but don't want to do it within the app (rnote here). From my perspective, I like to use my personal organization tool, and launch rnote on demand. Being fast, working with standard file formats pdf/svg is the most important part.

For organizing I personally use terminal file managers like ranger or lf. Notebook>Dividends>Tags>Pages are just directories even within onenote internal file format. One can free her/him from this and just rely on its file system and various tools around it. I think [your favorite] file manager could do many of such feature requests. Instead, I like to see more improvement on the core ideas which matter most.

I'm kind of advocating POSIX philosophy here

**Reply by flxzt** [reply]
You'll notice that the workspace functionality is just an integrated file browser, using regular files in user defined directories. I very intentionally did not use a database or a hidden, app-managed directory, because I feel the same as you and am sceptical about abstracting files and folders away.

So think of the workspace functionality as bookmarks with some added fluff like a custom name, icon and color. The app works just as good when files are launched from external file managers. :)
